isis/src/base/apps/phocube/main.cpp

```diff
@@ -205,20 +205,19 @@ void IsisMain() {
   if (subSolarGroundAzimuth) name += "Sub Solar Ground Azimuth";
   if (morphologyRank) name += "Morphology Rank";
   if (albedoRank) name += "Albedo Rank";
-  if (ra) name += "Right Ascension";
-  if (declination) name += "Declination";
   if (bodyFixedX) name += "Body Fixed X";
   if (bodyFixedY) name += "Body Fixed Y";
   if (bodyFixedZ) name += "Body Fixed Z";
-
+  if (ra) name += "Right Ascension";
+  if (declination) name += "Declination";
```

Comment:
Why have the bands been re-ordered? This could have down-stream impacts on people's existing processing tools that depend upon explicit band number output from phocube. IE, stats from=phoed.cub+4 may have different output now.

Reply:
Before we write values to each band, we check to see if the pixel is off body. If it is, then we used to right out NULL on each band. If it is on the planet, then we assign the pixel value for the corresponding keyword band by band. It was easier to move the RA and DEC bands to the end and outside of the if else clauses to check if we are off body than to refactor how the code previously wrote to each band.

Reply:
Now thinking about it, there might be a way that does not require that much refactoring. We would just have to keep track of which bands RA and DEC are and if the pixel is off body, then skip to the RA and DEC bands, write out the values to those bands, and write out NULL for the rest of the bands.

Reply:
Scripts have a tendency to use band numbers instead of searching for the correct band name and then getting the corresponding band number. Moving these band will undoubtedly break some scripts. I suggest adding the necessary code to keep the bands in the original order.